components/Util.js

```diff
@@ -71,9 +71,13 @@ export const sortByOrderAsc = (first, second) => first[1].order - second[1].orde
 export const isNumber = (str) => {
   return !isNaN(str);
 }
-  
-
 
+export const findContentTypeName = (arr,slugName) => (
+  arr.find((obj)=> (
```

Comment:
If there is only one parameter inside the `find` function, you don't have to put parenthesis.
```suggestion
  arr.find( obj => (
```